pom.xml

```diff
@@ -92,8 +98,39 @@
 			<artifactId>aws-java-sdk-sts</artifactId>
 			<version>${amazonaws.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>io.rest-assured</groupId>
+			<artifactId>rest-assured</artifactId>
+			<version>5.3.1</version>
```

Comment:
You are still using version directly on this line, use property instead.
You defined properties on line 28 for this then don't use 5.3.1 directly here, refer to line 99 to understand how to use properties.